templates/kms.yaml

```diff
@@ -48,6 +48,7 @@ Resources:
                   - - 'arn:aws:iam::'
                     - !Ref AWS::AccountId
                     - ':root'
+                - !Sub 'arn:aws:iam::${AWS::AccountId}:role/aws-service-role/ecs.amazonaws.com/AWSServiceRoleForECS'
```

Comment:
this is saying every ECS service role has access to this key?  what if we have other ECS services running in our account?

Reply:
looking at this again.  i believe this is wrong.  you don't want to give this role access to the key.  you want to give the ECS instances or tasks access to the key.  You can do that by doing the following..
1. taking the output of this stack (KmsKeyArn)
2. give EC2ServiceRole (in templates/gitlab-runner.yaml) role access to KmsKeyArn resource.
3. apply that role to the ECS instance or task.